wiki/Installation.md

```diff
@@ -14,29 +16,30 @@ Add the following line to the package dependencies in `Package.swift`:
 
 ## Get the correct product
 
-### Add one one of the following products to your target dependencies.
+Add one one of the following products to your target dependencies.
 
-#### If you want to use Workflow with UIKit
+#### __If you want to use Workflow with UIKit__
```

Comment:
Are we using __ or ** in other files?  Is this something we should have a preference on?

Reply:
We are using ** elsewhere and that's what GitHub does by default as well, so I will replace.